Content.Server/Administration/Commands/ChangeVisibilityCommand.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Content.Shared.Administration;
+using Robust.Server.GameObjects;
+using Robust.Shared.Console;
+
+namespace Content.Server.Administration.Commands;
+
+[AdminCommand(AdminFlags.Admin)]
+public sealed class ChangeVisibilityCommand : LocalizedCommands
```

Comment:
Make it a ToolshedCommand, the conversion is normally pretty simple once you look at usages in the codebase and the (incomplete) docs but do feel free to ask any question